packages/camel-catalog/catalog-generator/pom.xml

```diff
@@ -0,0 +1,84 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>io.kaoto</groupId>
+    <artifactId>kaoto-camel-catalog-parent</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+    <relativePath>../pom.xml</relativePath>
+  </parent>
+  <artifactId>catalog-generator</artifactId>
+  <packaging>jar</packaging>
+  <name>catalog-generator</name>
+  <description>
+    A Camel Catalog generator for Kaoto.
+  </description>
+  <url>https://kaoto.io</url>
+
+  <dependencies>
+    <!-- Camel dependencies -->
+    <dependency>
+      <groupId>org.apache.camel</groupId>
+      <artifactId>camel-catalog</artifactId>
+      <version>${version.camel}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.camel</groupId>
+      <artifactId>camel-catalog-maven</artifactId>
+      <version>${version.camel}</version>
+    </dependency>
+
+    <dependency>
+      <groupId>io.fabric8</groupId>
+      <artifactId>kubernetes-model</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-api</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-engine</artifactId>
+      <scope>test</scope>
+    </dependency>
```

Comment:
JUnit project is recommending to use a single depdency:

```
<dependency>
        <groupId>org.junit.jupiter</groupId>
        <artifactId>junit-jupiter</artifactId>
        <version>5.10.2</version> <!-- can be omitted when using the BOM -->
        <scope>test</scope>
    </dependency>
```
https://junit.org/junit5/docs/current/user-guide/#running-tests-build-maven-engines-configure

Reply:
How do I know if I'm using the BOM?

Reply:
need to look to all depdencyManagement of this module and its parents.

And the junit bom is not used, there are direct JUnit dependencies specified https://github.com/KaotoIO/kaoto/blob/a3c1b53166413386d6670f7153d64deb451bfa25/packages/camel-catalog/pom.xml#L105-L115 (that would be better to replace by the junit bom declaration)

Reply:
Ok, I see, in any case, the intention is to replace the existing structure with the "~new~ rearranged code", so we should end up with a single maven project I think, we can tackle this once we're on that journey (it's the next step of this PR)